demo/blocks_inputs/run.py

```diff
@@ -12,14 +12,14 @@ def mirror(x):
     txt = gr.Textbox(label="Input", lines=2)
     txt_2 = gr.Textbox(label="Input 2")
     txt_3 = gr.Textbox("", label="Output")
-    btn = gr.Button("Submit")
```

Comment:
In the original issue, the discord user said he couldn't find demos using `value` despite it being used in demos already. 

I figured that was because we didn't set it explicitly with a keyword argument so I went through some demos and set value via keyword arg.

Reply:
Good idea